lib/curl_threads.c

```diff
@@ -35,7 +35,9 @@
 #endif
 
 #include "curl_threads.h"
+#ifdef BUILDING_LIBCURL
```

Comment:
Why is this necessary? the header itself has an `#ifdef CURLDEBUG`, should that not be enough?

Reply:
Because I added curl_threads.c as a source of libtest, the memory wrappers in curl_memory.h are non-public symbols that libtest can't link against in shared library mode. CURLDEBUG is defined in libtest as well.